src/Compilers/CSharp/Portable/Binder/Binder_Query.cs

```diff
@@ -648,7 +659,10 @@ private UnboundLambda MakeQueryUnboundLambda(RangeVariableMap qvm, ImmutableArra
         {
             return MakeQueryUnboundLambda(expression, new QueryUnboundLambdaState(this, qvm, parameters, (LambdaSymbol lambdaSymbol, Binder lambdaBodyBinder, DiagnosticBag diagnostics) =>
             {
-                return lambdaBodyBinder.BindLambdaExpressionAsBlock(expression, diagnostics);
```

Comment:
I just saw the new `RefQueryClause` test below. Looks like calling `SkipRef` here was un-necessary. Is that right?